[2024-11-28T20:30:04.388+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-28T20:30:04.483+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Fetch_Load.fetch_historical_weather scheduled__2024-11-27T20:30:00+00:00 [queued]>
[2024-11-28T20:30:04.507+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Fetch_Load.fetch_historical_weather scheduled__2024-11-27T20:30:00+00:00 [queued]>
[2024-11-28T20:30:04.508+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-28T20:30:04.536+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_historical_weather> on 2024-11-27 20:30:00+00:00
[2024-11-28T20:30:04.635+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1261) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-28T20:30:04.636+0000] {standard_task_runner.py:63} INFO - Started process 1272 to run task
[2024-11-28T20:30:04.659+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Weather_Fetch_Load', 'fetch_historical_weather', 'scheduled__2024-11-27T20:30:00+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/weather-dag.py', '--cfg-path', '/tmp/tmpwm17_biy']
[2024-11-28T20:30:04.732+0000] {standard_task_runner.py:91} INFO - Job 242: Subtask fetch_historical_weather
[2024-11-28T20:30:05.172+0000] {task_command.py:426} INFO - Running <TaskInstance: Weather_Fetch_Load.fetch_historical_weather scheduled__2024-11-27T20:30:00+00:00 [running]> on host fd43cd87157f
[2024-11-28T20:30:05.538+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Fetch_Load' AIRFLOW_CTX_TASK_ID='fetch_historical_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-11-27T20:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-27T20:30:00+00:00'
[2024-11-28T20:30:05.545+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-28T20:30:06.246+0000] {python.py:237} INFO - Done. Returned value was:             Date  temperature_2m_max  ...  windspeed_10m_max  avg_temperature
0     2022-01-01                 8.0  ...               18.0             2.55
1     2022-01-02                 7.0  ...               11.8             1.35
2     2022-01-03                 5.5  ...               11.4             0.55
3     2022-01-04                 4.1  ...               11.7             1.30
4     2022-01-05                 4.8  ...               11.2             0.15
...          ...                 ...  ...                ...              ...
1058  2024-11-24                11.1  ...               15.2             9.10
1059  2024-11-25                 7.7  ...               24.5             2.80
1060  2024-11-26                 2.8  ...               26.4            -0.40
1061  2024-11-27                 NaN  ...                NaN              NaN
1062  2024-11-28                 NaN  ...                NaN              NaN

[1063 rows x 7 columns]
[2024-11-28T20:30:06.295+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-28T20:30:06.502+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Weather_Fetch_Load, task_id=fetch_historical_weather, run_id=scheduled__2024-11-27T20:30:00+00:00, execution_date=20241127T203000, start_date=20241128T203004, end_date=20241128T203006
[2024-11-28T20:30:06.537+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-28T20:30:06.567+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-28T20:30:06.573+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
